src/stations.rs

```diff
@@ -6,7 +6,7 @@ use crate::model;
 #[derive(Debug, PartialEq)]
 pub struct Station {
     pub name: String,
-    pub coords: model::Coordinates,
+    pub coordinates: model::Coordinates,
```

Comment:
I like this renaming!

Reply:
But I also wonder if `Station` wants to be in `model`? I'm not sure what our ideal module structure is yet, but this is fine for now. Since we're using `Coordinates` all over the place, we should probably `use` it directly so we don't need to be referencing `model::` so often.

Reply:
My thought was to try to keep the API for querying stations in a module with a pretty small public API. That way any changes to how we load, who we load from, how we store data, etc. don't impact the rest of the code base. Keep `Station` in there made it clear that it's owned by that module.

Not married to that scheme if you want to organize it differently though. 🤷‍♂

Reply:
Totally makes sense, let's keep it in here.